frontend/app/controllers/create-badges.js

```diff
@@ -34,6 +38,25 @@ export default Controller.extend({
           _this.set('userError', userErrors);
         }
       });
+
+      let fontColorCode = '#' + _this.defFontColor;
+      let badgeRecord = _this.get('store').createRecord('badge', {
+        csv        : _this.csvFile,
+        image      : _this.custImgFile,
+        font_color : fontColorCode,
+        uid        : _this.uid,
+        badge_size : 'A3',
+        font_size  : '20',
```

Comment:
why is `font_size` hardcoded here? According to the demo, you said variable font-size is active

Reply:
Waiting for the PR of enhanced font slider to be merged so that I can implement the changes. As he has provided the working solution for the component, so I don't think it is right to do his work in my PR. As soon as his PR gets merged I will add the change.